File: TenMinsPy/day60.py
# Check if string is 8 digits or 11 or starting with +, plus 12 digits (+63)

class day60:
    def checkNum(self, s):
        if s[0] == '+':
            s = s[1:]
            if len(s) == 12 and s.isdigit():
                print('nice')
            else:
                print('nah')
        else:
            if len(s) in (8,11) and s.isdigit():
                print('nice')
            else:
                print('nah')

day = day60()   
s = input('enter num\n')
day.checkNum(s)

File: TenMinsJava/day60.java
// Check if string is 8 digits or 11 or starting with +, plus 12 digits (+63)
package TenMinsJava;

import java.util.Scanner;

public class day60 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.err.println("Enter phone number");
        String s = scan.nextLine();
        checkNum(s);
    }

    public static void checkNum(String s) {
        if (s.length() == 8) {
            if (s.matches("\\d+")) { 
                System.err.println("nice"); 
            }
            else { 
                System.err.println("nah"); 
            }
        }
        if (s.length() == 11){
            if (s.matches("\\d+")) { 
                System.err.println("nice"); 
            }
            else {
                System.err.println("nah"); 
            }
        }
        if (s.charAt(0) == '+' && s.length() == 13){
            s = s.substring(1);
            if (s.matches("\\d+")) {
                System.err.println("nice"); 
            }
            else {
                System.err.println("nah"); 
            }
        }
        else {
            System.err.println("nah bro");
        }
    }
}